_im-plugin/index-transforms/transforms-apis.md

```diff
@@ -128,17 +128,17 @@ You can specify the following options in the HTTP request body:
 Option | Data Type | Description | Required
 :--- | :--- | :--- | :---
 enabled | Boolean | If true, the transform job is enabled at creation. | No
-schedule | JSON | The schedule the transform job runs on. | Yes
+schedule | Object | The schedule the transform job runs on. | Yes
 start_time | Integer | The Unix epoch time of the transform job's start time. | Yes
 description | String | Describes the transform job. | No
 metadata_id | String | Any metadata to be associated with the transform job. | No
 source_index | String | The source index whose data to transform. | Yes
 target_index | String | The target index the newly transformed data is added into. You can create a new index or update an existing one. | Yes
-data_selection_query | JSON | The query DSL to use to filter a subset of the source index for the transform job. See [query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl) for more information. | Yes
-page_size | Integer | The number of fields to transform at a time. Higher number means higher performance but requires more memory and can cause higher latency. (Default: 1) | Yes
+data_selection_query | Object | The query DSL to use to filter a subset of the source index for the transform job. See [query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl) for more information. | Yes
+page_size | Integer |  The maximum number of buckets OpenSearch can use to transform data. Higher number means higher performance but requires more memory and can cause higher latency. | Yes
```

Comment:
This is a bit misleading as it seems to say that it limits the number of buckets which can be transformed, which isn't right. What actually occurs is that internally, in each transform job run, we transform all of the buckets, and the page size determines how many buckets we index at a time as we batch through the produced buckets.